charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -136,6 +138,8 @@ spec:
             defaultMode: 420
             secretName: webhook-server-cert
         {{- end }}
+        - name: tmp
+          emptyDir: {}
```

Comment:
How is it going to work? An empty volume is mapped to /tmp while another non-empty volume is mapped to /tmp/admiconfig? Won't there be a collision?

Reply:
https://github.com/fybrik/fybrik/blob/304ac036a2b9f8ac3beea2b2a1dbd48d742f09db/manager/Dockerfile#L4-L8
According to this, /tmp is the root directory of the manager pod. On the other hand, we want to be able to create files in /tmp. I'd say it's going to be a problem to mark /tmp as read-only.

Reply:
No, the settings above mean that `/tmp` is the home directory for the manager user and it is the running directory.
We don't want to mark `/tmp` as read-only, opposite we want to set all filesystem except `/tmp` as read-only.

Reply:
OK. Still, to avoid any collisions now and in the future, let's replace /tmp with /data (including policies and taxonomy), and allocate /tmp as a new empty volume.

Reply:
currently, the path to policies and taxonomy is hard-coded to `/tmp/adminconfig` and `/tmp/taxonomy`.  Does it make sense to insert the path to helm values with the default prefix in `data`?

Reply:
yes, please do it, and defined a Pod environment variable with the value. So we will be able to define them in a single place

Reply:
Why would we want to add an unnecessary configuration? There is a chance that a "wrong" value will cause problems in the deployment, which will be very hard to debug.

Reply:
@shlomitk1 I don't agree with you about hardcoded configurations. Why do you think that env variables create problems for deployments?

Reply:
> @shlomitk1 I don't agree with you about hardcoded configurations. Why do you think that env variables create problems for deployments?

Env variables don't. Extra helm values can present a problem. We are trying to avoid /tmp being used for "data" files, so why let anyone to change the default prefix to something we don't want? 
It's fine to let users configure in which namespace they want to run modules, which connectors to deploy, etc., but the file structure of the pod is internal and I don't see a reason to let anyone change it.